ios/Podfile.lock

```diff
@@ -105,6 +105,12 @@ PODS:
     - ExpoModulesCore
   - EXScreenCapture (5.3.0):
     - ExpoModulesCore
+  - EXSensors (12.3.0):
+    - ExpoModulesCore
+  - EXSplashScreen (0.20.5):
```

Comment:
This splash screen keeps making a comeback 😆

Reply:
Yeah I decided not to remove it, as it seems that it's a default :/